test/acceptance/features/bindAppToServiceAnnotations.feature

```diff
@@ -427,3 +427,73 @@ Feature: Bind an application to a service using annotations
                 ready: true
             """
         Then Secret "binding-request-backend-ann" is empty
+
+
+    @disabled
```

Comment:
can we enable to test or it is not working?

Reply:
Okay, I'll enable it to see failure then will post the CI output and will disable the test again.